server/journeys/referral/details/view.njk

```diff
@@ -1,7 +1,7 @@
 {% from "govuk/components/date-input/macro.njk" import govukDateInput %}
 {% from "govuk/components/select/macro.njk" import govukSelect %}
 {% from "govuk/components/button/macro.njk" import govukButton %}
-{% from "hmpps/components/date-picker/macro.njk" import hmppsDatePicker %}
+{% from "moj/components/date-picker/macro.njk" import mojDatePicker %}
```

Comment:
the js import below:
```
{% block additionalScripts %}
    <script src="/assets/javascript/datepicker.js"></script>
{% endblock %}
```
was added to support `hmppsDatePicker`
does `mojDatePicker` need something like that, or can this js import be removed?

Reply:
Yes these, and the ones in the other three views the date picker is in can have this block removed

Reply:
It's only in this one view that I can find?

Reply:
I found these in the branch
![image](https://github.com/user-attachments/assets/5c331cc7-ced4-4f28-b755-67f48de3db98)

Reply:
I merged those two together as part of 334 here https://github.com/ministryofjustice/hmpps-challenge-support-intervention-plan-ui/pull/73/files

Reply:
You might be looking at an old view of the code?

Reply:
They were in /dist! Sorry